python/feathub/dsl/ast.py

```diff
@@ -234,12 +225,8 @@ def __init__(self, func_name: str, args: ArgListNode) -> None:
 
     def eval_dtype(self, variable_types: Dict[str, DType]) -> DType:
         arg_types = [arg.eval_dtype(variable_types) for arg in self.args.values]
-        function_type_evaluator = FUNCTION_RESULT_TYPE_EVALUATOR.get(self.func_name)
-        if function_type_evaluator is None:
-            raise RuntimeError(
-                f"Unknown result type evaluator for function {self.func_name}."
-            )
-        return function_type_evaluator(arg_types)
+        builtin_func = BuiltInFunc(self.func_name)
```

Comment:
Instead of constructing an instance of `BuiltInFunc` by calling its constructor, would it be more intuitive to use a static method such as `BuiltInFunc#get(func_name)`?